kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/config/applyConfigFromAbstractProjectConfig.kt

```diff
@@ -42,6 +42,7 @@ internal fun applyConfigFromProjectConfig(config: AbstractProjectConfig, configu
    // discovery
    config.discoveryClasspathFallbackEnabled?.let { configuration.discoveryClasspathFallbackEnabled = it }
    config.disableTestNestedJarScanning?.let { configuration.disableTestNestedJarScanning = it }
+   config.includePrivateClasses?.let { configuration.ignorePrivateClasses = it }
```

Comment:
Include vs ignore

I think at some point it got inverted

Reply:
Thanks.